controllers/zora/clusterscan_controller.go

```diff
@@ -111,10 +112,9 @@ func (r *ClusterScanReconciler) reconcile(ctx context.Context, clusterscan *v1al
 	}
 
 	if !cluster.Status.ConditionIsTrue(v1alpha1.ClusterReady) {
-		err := errors.New(fmt.Sprintf("the Cluster %s is not Ready", cluster.Name))
-		log.Error(err, "Cluster is not ready")
-		clusterscan.SetReadyStatus(false, "ClusterNotReady", err.Error())
-		return err
+		notReadyErr = errors.New(fmt.Sprintf("the Cluster %s is not Ready", cluster.Name))
```

Comment:
here can be shortened by using `fmt.Errorf()`:
```suggestion
		notReadyErr = fmt.Errorf("the Cluster %s is not Ready", cluster.Name)
```